### DISCUSSION TITLE
Rework Reduction Gear

### DISCUSSION THREAD

**Discussion by TBN-MapleWheels** [PeasantFoundry/Mechtrauma#96] [General]
- Drive Gears need to cost more.
- Add a "Gear Assembly" component that only takes damage once all other components have reached 0 durability and the Reduction Gear World Item has hit 0 durability.
- Upon failure of the Gear Assembly, create a Fragmentation Explosion, with fragments causing massive damage.
- Gear Assembly should be very expensive to repair.

**Comment by PeasantFoundry** [comment]
A near term rework of the reduction gear has been completed and pushed. The active fail state has been changed from fire to a frag explosion and the drive gears wear overtime with damage increasing on a curve as the item condition decreases. For the moment the item condition is being treated as the gear assembly/ gear housing. 

I also fixed a few bugs where oil consumption and filter deterioration weren't calculating correctly.
